src/content/Angular-CSharp/ClientApp/proxy.conf.js

```diff
@@ -1,7 +1,7 @@
 const { env } = require('process');
 
 const target = env.ASPNETCORE_HTTPS_PORT ? `https://localhost:${env.ASPNETCORE_HTTPS_PORT}` :
-  env.ASPNETCORE_URLS ? env.ASPNETCORE_URLS.split(';')[0] : 'http://localhost:8080';
+  env.ASPNETCORE_URLS ? env.ASPNETCORE_URLS.split(';')[0] : 'https://localhost:5001';
```

Comment:
I switched this from the IIS to the Kestrel default port when there's not environment variable from launch settings since Kestrel is now preferred. We use HTTPS by default currently when reading env variables.